.github/workflows/maven-deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - Issue-2134-ossrh-to-central-portal-migration
```

Comment:
Can you remove this line?

Reply:
Fixed by commit e593bac1. Sorry, it wasn't pushed.